src/commands/package.js

```diff
@@ -57,7 +57,7 @@ export default asyncCommand({
 		//Create the xml descriptor file for the zimlet
 		let xmlFile = `${argv.name}.xml`;
 
-		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${argv.description}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
+		let zimletXML = `<zimlet name="${argv.name}" version="${argv.pkgVersion}" description="${argv.description}" label="${argv.displayName}" zimbraXZimletCompatibleSemVer="${argv.zimbraXVersion}">`;
```

Comment:
argv.label

Reply:
we should also update help instruction to show that this parameter is supported

Reply:
https://github.com/Zimbra/zimlet-cli/pull/55/files#diff-ee598ee5680e1c35974febfe40adc27fL12